packages/remix-server-runtime/interface/README.md

```diff
@@ -0,0 +1,16 @@
+# Remix Server Runtime Interface
+
+To be a Remix Server Runtime, a package must export:
+- `createCookie`
+- `isCookie`
+- `createSession`
+- `isSession`
+- `createSessionStorage`
+- `createCookieSessionStorage`
+- `createMemorySessionStorage`
+- `json`
+- `redirect`
+
+Corresponding types for each of these exports are defined in this directory.
+
+Additionally, each Remix Server Runtime must re-export the type definitions in `./types.ts`.
```

Comment:
I think you meant to say `./index.ts`

Reply:
Oops, I haven't added the `./types.ts` file to the draft yet. Coming soon...